src/PointwiseFunctions/ScalarGaussBonnet/ScalarMomentum.cpp

```diff
@@ -0,0 +1,30 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "PointwiseFunctions/ScalarGaussBonnet/ScalarMomentum.hpp"
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/Tensor/EagerMath/DotProduct.hpp"
+#include "Elliptic/Systems/ScalarGaussBonnet/Tags.hpp"
+#include "Elliptic/Systems/Xcts/Tags.hpp"
+#include "Evolution/Systems/CurvedScalarWave/Tags.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Tags.hpp"
+
+namespace sgb {
+
+void compute_pi(const gsl::not_null<Scalar<DataVector>*> result,
```

Comment:
Remove `compute_` prefix. Name `scalar_momentum` if you think `pi` is too short.